data/sql/updates/pending_db_world/rev_1555926066020418100.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1555926066020418100');
+
+-- Remove UNIT_FLAG_IMMUNE_TO_PC from NPC 'Ethereum guardian'
+UPDATE `creature_template` SET `unit_flags`='0' WHERE `entry`=20854;
```

Comment:
just a note: this is not the best way to deal with flags, but it's OK to merge this since it has already been tested and approved.

However I've created a TODO in the wiki so we don't forget to do this: https://github.com/azerothcore/wiki/issues/39